package.json

```diff
@@ -1336,14 +1341,23 @@
               "request": "attach",
               "processId": "^\"\\${command:pickRemoteProcess}\"",
               "pipeTransport": {
-                "pipeCwd": "^\"\\${workspaceRoot}\"",
+                "pipeCwd": "^\"\\${workspaceFolder}\"",
                 "pipeProgram": "^\"${1:enter the fully qualified path for the pipe program name, for example '/usr/bin/ssh'}\"",
                 "pipeArgs": [],
                 "debuggerPath": "^\"${2:enter the path for the debugger on the target machine, for example ~/vsdbg/vsdbg}\""
               }
             }
           }
-        ]
+        ],
+        "windows": {
+          "program": "./.debugger/vsdbg-ui.exe"
+        },
+        "osx": {
+          "program": "./.debugger/vsdbg-ui"
+        },
+        "linux": {
+          "program": "./.debugger/vsdbg-ui"
+        }
```

Comment:
I'm pretty sure that you don't want these changes in your PR. Is that correct @gregg-miskelly?

Reply:
It can happen when debugging the extension.